docs/features/execution_payload_retention.md

```diff
@@ -12,19 +12,37 @@ This allows access via the API (or optionally direct DB/Elasticsearch querying)
 
 ## Payload record cleanup
 
-To reduce storage requirements, a CloudWatch rule (`{stack-name}-dailyExecutionPayloadCleanupRule`) triggering a daily run of the provided cleanExecutions lambda has been added.  This lambda will remove all 'completed' and 'non-completed' payload records in the database that are older than the specified configuration.
+<!-- To reduce storage requirements, a CloudWatch rule (`{stack-name}-dailyExecutionPayloadCleanupRule`) triggering a daily run of the provided cleanExecutions lambda has been added.  This lambda will remove all 'completed' and 'non-completed' payload records in the database that are older than the specified configuration. -->
 
 ### Configuration
 
-The following configuration flags have been added.  They may be overridden in your deployment configuration by adding the appropriate keys:
+The following configuration flags have been made available in the `cumulus` module.   They may be overridden in your deployment's instance of the `cumulus` module by adding the following configuration options:
 
-- complete_execution_payload_timeout
+#### daily_execution_payload_cleanup_schedule_expression _(string)_
 
-This flag defines the cleanup threshold for executions with a 'complete' status in days.   Records with updateTime values older than this with payload information  will have that information removed.
+This configuration option sets the execution times for this lambda to run, using a Cloud Watch cron expression.
 
-Default value is 10 days.
+Default value is `"cron(0 4 * * ? *)"`.
 
-- non_complete_execution_payload_timeout
+#### complete_execution_payload_timeout_disable _(bool)_
+
+This configuration option, when set to true, will disable all cleanup of `completed` execution payloads.
+
+Default value is `false`.
+
+#### complete_execution_payload_timeout _(number)_
+
+This flag defines the cleanup threshold for executions with a 'completed' status in days.   Records with updateTime values older than this with payload information  will have that information removed.
+
+Default value is `10`.
+
+#### non_complete_execution_payload_timeout_disable _(bool)_
+
+This configuration option, when set to true, will disable all cleanup of "non-complete" (any status _other_ than `completed`) execution payloads (e.g. ).
```

Comment:
Should there be something after "(e.g."?

Reply:
No, e.g. should be removed.

Reply:
bccde3043